templates/home.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Why do you need this file?

Reply:
The file is useless now. I used it to see content display in browser.Will be dropped in the next  changes.